documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -2288,21 +2288,21 @@ The following would be invalid:
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [ARM](#arm-violations): [Errors](#arm-errors) or [Warnings](#arm-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
 
-### <a name="r4006" ></a>R4006 XmsCodeGenerationSettingDeprecated
+### <a name="r4006" ></a>R4006 DeprecatedXmsCodeGenerationSetting
 
 **Category** : ARM Warning
 
-**Applies to** : ARM OpenAPI(swagger) specs
+**Applies to** : ARM and Data plane OpenAPI(swagger) specs
 
-**Output Message** : The x-ms-code-generation-settings extenison was deprecated. Consider remove it from the swagger.  
+**Output Message** : The x-ms-code-generation-setting extension is being deprecated. Please remove it and move settings to readme file for code generation.
 
-**Description** : New autorest (V3) will stop supporting “x-ms-code-generation-settings” inside the Swagger. This was designed before there was Readme.md to hold configuration options, and this node should disappear and be replaced by actual readme descriptions.
+**Description** : x-ms-code-generation-settings is being deprecated. AutoRest (v3) will stop supporting it inside the Swagger. There was Readme.md to hold the same configuration options. Please ensure to remove the parameter from swagger spec and move settings to readme.
 
 **CreatedAt**: March 18, 2020
 
 **LastModifiedAt**: March 18, 2020
 
-**How to fix the violation**: The only value of this node today is to override the client name, which could be done with a “title” line in the Readme, and some C# only options that could be done readme.csharp.me (and those options disappears anyway with the new codegen for Csharp).
+**How to fix the violation**: Since the only value of this extension today is to override the client name, which could be done with a “title” line in the Readme,you could remove the extension from swagger spec and move settings to readme.
```

Comment:
this seems clear to me.  "...which could be done with a 'title' line in readme file..." better?